dataflows/processors/printer.py

```diff
@@ -26,8 +27,15 @@ def _table_print(data, kwargs):
         print(data)
 
 
+def truncate_cell(value, max_size):
+    if sys.getsizeof(value) > max_size:
+        return str(value)[:max_size] + ' ...'
```

Comment:
why check the byte size and not the actual length of str(value)?

Reply:
value is not necessarily a string, I thought it's better not to convert to str, especially if it's a very large value